.editorconfig

```diff
@@ -3,3 +3,7 @@ root = true
 [*.rb]
 indent_style = space
 indent_size = 2
+
+[*.js]
+indent_style = space
+indent_size = 2
```

Comment:
I don't think this is required in the file tree and should be added to the `.gitignore`

Reply:
I fixed this myself.

Reply:
@thealphadollar No, this _is_ required. `.editorconfig` ensures that everyone follows the same indenting. I will remove it from `.gitignore` and add it back in another PR.